### ISSUE TITLE
NPM package isn't up to date and uses lodash version with a "critical" vulnerability

### ISSUE THREAD

**Issue by jaredkrinke** [chiefy/metalsmith-express#9]
This repository shows that the lodash dependency was removed, but the version of [metalsmith-express](https://www.npmjs.com/package/metalsmith-express) on NPM appears to be out of date. Additionally, it depends on a version of lodash with a vulnerability that's reported as "critical":

```txt
$ npm audit
...
  Critical        Prototype Pollution in lodash                                 

  Package         lodash                                                        

  Patched in      >=4.17.12                                                     

  Dependency of   metalsmith-express                                            

  Path            metalsmith-express > lodash                                   

  More info       https://github.com/advisories/GHSA-jf85-cpcp-j695       
```

My understanding is that you're the only one who can update the NPM package, so... pretty please? :)

My work around will be to clone the repository, create a tag, and then reference that tag in my project.

Thanks for building this plugin!

**Comment by chiefy** [comment]
Sure, I will try to tackle this ASAP, thanks for pinging!

**Comment by jaredkrinke** [comment]
Are you the owner of the [published NPM package](https://www.npmjs.com/package/metalsmith-express) as well? That was actually the issue I ran into (my tool uses the published package, which appears to be out of date, and gets flagged for the vulnerability above).

But, wow, thanks for being super responsive! That was very quick :)

**Comment by chiefy** [comment]
Whoops, sorry forgot about that - [published 1.0.0](https://www.npmjs.com/package/metalsmith-express)

**Comment by jaredkrinke** [comment]
Looks good now, thanks!
